docs/source/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- Project information -----------------------------------------------------
+
+project = 'git-machete'
+copyright = '2017-2021, Paweł Lipski'
+author = 'Paweł Lipski'
+
+# The full version, including alpha/beta/rc tags
+release = '3.3.0'
```

Comment:
Hmmm this needs to be kept in sync with the software version itself... pls check if there is any neat way to keep it in line

Reply:
(otherwise, probably some shell script executed from CI FTW)

Reply:
How about that? This should be fine, because depending on which branch docs will be based on it will read the latest release notes and put such label on docs.
``` python
def get_last_version_from_release_note():
    with open("../../RELEASE_NOTES.md") as file:
        text = file.read()
    version_pattern = re.compile(r"[0-9]\.[0-9]\.[0-9]")
    return version_pattern.search(text).group()


release = get_last_version_from_release_note()
```

Reply:
Piotr has proposed `release = git_machete.__version__` instead

Reply:
Yeah, if possible, `__version__` FTW